css/layout.css

```diff
@@ -31,4 +35,16 @@
   .column-half {
     width: 50%;
   }
+
+  .notes {
+    margin-left: -105%;
+  }
```

Comment:
**_Holy schnikies_**. Why is this necessary? You should find a different way. Values above 100% are really bad in general, and negative margins are equally bad.